### DISCUSSION TITLE
Enable another domain or add new domain in an existing application

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11566] [Q&A]
Folks, please advise on below request: 

Enable another domain in an existing domain to used the multinancy service please advise how i do like it is clone one of existing application.

**Comment by Skrypt** [comment]
Just add the domain in the tenant settings and it should work. Of course you need to also set Azure or IIS or Nginx to accept this domain.

**Reply by saleemhaq** [reply]
> Just add the domain in the tenant settings and it should work. Of course you need to also set Azure or IIS or Nginx to accept this domain.

and it points to the same directory as the main application point directory?

**Reply by saleemhaq** [reply]
> > Just add the domain in the tenant settings and it should work. Of course you need to also set Azure or IIS or Nginx to accept this domain.
> 
> and it points to the same directory as the main application point directory?

Also there is possibility of: we have custom page required with separate admin to update content of content type
all together different application?

**Reply by ns8482e** [reply]
Are you asking to have multiple domain in same tenant or you really asking for multiple tenant with a domain?

**Comment by saleemhaq** [comment]
1- Are you asking to have multiple domain in same tenant
 yes?

 2- You really asking for multiple tenant with a domain?
 I want to say I create sub domain and sub domain point the main directory
of domain?

Please advise on above.

**Reply by ns8482e** [reply]
> 1- Are you asking to have multiple domain in same tenant 
> yes?

you can provide comma separated hosts 

![image](https://user-images.githubusercontent.com/23270244/164484203-e95e89bf-4f53-457b-9517-dae539139796.png)
